### ISSUE TITLE
connection error when vhost is specified as part of the url

### ISSUE THREAD

**Issue by scarmuega** [mateodelnorte/servicebus#53]
The amqp uri specification shows how to specify a vhost within the url. Example:

`amqp://guest:guest@myrabbit.net/myvhost`

But, if we use the following code to connect with servicebus...
```
var bus = require('servicebus').bus({ 
    url: 'amqp://guest:guest@myrabbit.net/myvhost',
});
```
... we'll get an ECONNRESET error.

Browsing the code I found that the underlying `amqplib` supports this kind of URIs, but `servicebus` is expecting to receive the vhost in a different property in the options object. If the user doesn't specify a vhost, it will append a trailing slash automatically, which ends up generating a wrong URI.

I think that `servicebus` shouldn't modify the URI and just pass it through to the `amqplib`.

**Comment by mateodelnorte** [comment]
Sounds good to me. Wanna spin up a pull request?

**Comment by mateodelnorte** [comment]
fixed by #57
